fluentd/fluent-plugin-grafana-loki/lib/fluent/plugin/out_loki.rb

```diff
@@ -67,6 +74,13 @@ def configure(conf)
 
         @label_keys = @label_keys.split(/\s*,\s*/) if @label_keys
         @remove_keys = @remove_keys.split(',').map(&:strip) if @remove_keys
+
+        @cert = OpenSSL::X509::Certificate.new(File.read(@cert)) if @cert
+        @key = OpenSSL::PKey::RSA.new(File.read(key)) if @key
```

Comment:
Any chance to use `OpenSSL::PKey.read`?

ref: https://ruby-doc.org/stdlib-2.4.0/libdoc/openssl/rdoc/OpenSSL/PKey.html#method-c-read

This method should be more flexible to handle private key.

Reply:
Should be possible, I'll update this in the next commit